src/main/java/com/snowflake/kafka/connector/internal/ingestsdk/KcStreamingIngestClient.java

```diff
@@ -60,14 +60,17 @@ public KcStreamingIngestClient(SnowflakeStreamingIngestClient client) {
    * <p>Any exceptions will be passed up, a sfexception will be converted to a connectexception
    *
    * @param streamingClientProps the properties for the client
+   * @param parameterOverrides Helps to override any default parameters in streaming client
    * @param clientName the client name to uniquely identify the client
    */
-  protected KcStreamingIngestClient(Properties streamingClientProps, String clientName) {
+  protected KcStreamingIngestClient(
+      Properties streamingClientProps, Map<String, Object> parameterOverrides, String clientName) {
     try {
       LOGGER.info("Creating Streaming Client: {}", clientName);
       this.client =
           SnowflakeStreamingIngestClientFactory.builder(clientName)
               .setProperties(streamingClientProps)
+              .setParameterOverrides(parameterOverrides)
```

Comment:
May be I am missing something, but we can always add more to `parameterOverrides` from the config.

Reply:
Correct, but that requires more changes in KC and a new release, right? I was thinking maybe we can just accept a json and pass to the SDK

Reply:
I see, accepting json makes sense but it just gives more control for configuration, which in past IIRC was not something we encouraged. For example flush time. 

We can add json once we really need more knobs. I will pass this time since we don't anticipate this configuration change. 

I though agree having flexibility to modify config without kc new release. 
So I will make that change to use bdex versions directly.